podium/talks/urls.py

```diff
@@ -1,7 +1,7 @@
 from django.conf.urls import url
-
 from . import views
 
 urlpatterns = [
     url(r'^submit/', views.submit_talk_view, name='talks-submit'),
+    url(r'^sessions/', views.session_table, name='talks-sessions'),
```

Comment:
Small issue here.  Looks like this is referencing a view function that doesn't exist or was renamed.